fish

```diff
@@ -0,0 +1,33 @@
+# All pitfalls are taken from https://fishshell.com/docs/current/faq.html
+
+# Creating a variable:
+set -x variable value # typically set -gx key value
```

Comment:
This isn't correct. Setting a variable is just `set var_name var_value`. `set -x` is to set an exported variable, which is rarely needed and probably not what you are looking for (and can cause other problems).

Reply:
Fixed. ;)